graph_annotation_test.go

```diff
@@ -73,6 +78,30 @@ func TestCreateGraphAnnotation(t *testing.T) {
 	if err != nil {
 		t.Error("err shoud be nil but: ", err)
 	}
+
+	if annotation.Service != "My-Blog" {
+		t.Error("request sends json including Service but: ", annotation.Service)
+	}
```

Comment:
It seems mackerel-client-go contains such error messages in several tests. I'll send another pull request for the modifications of the error message.